R/quosure.R

```diff
@@ -211,6 +217,13 @@ quo_label <- function(quo) {
 quo_text <- function(quo, width = 60L, nlines = Inf) {
   expr_text(quo_expr(quo), width = width, nlines = nlines)
 }
+#' @export
+quo_name <- function(quo) {
+  if (!is_symbol(quo)) {
+    abort("`quo` must quote a symbol")
```

Comment:
do we need a name that reflects better the intent behind that function? otherwise users might be confused between all the options: `_text()`, `_label()`, `_name()`. I'm not sure I understand what is this function for.

The `_text()` versions might be multi-line so you want something that fits on one line is that right? Should `expr_label()` be `expr_format()`, and then `quo_name()` would be `quo_label()` (and we'd have `expr_label()` version etc)?